src/app/views/river-detail/components/main-tab/components/beta-box/beta-box.vue

```diff
@@ -189,68 +190,72 @@ export default {
       river: state => state.RiverDetail.data,
       editMode: state => state.Global.editMode,
       gages: state => state.RiverGages.data?.gauges ?? [],
-      metrics: state => state.RiverGages.data?.metrics ??[]
+      metrics: state => state.RiverGages.data?.metrics ?? []
     }),
-    releases() {
-      return this.$store.getters['RiverEvents/releases']
+    releases () {
+      // the zero-index in this will be the last or next release.
+      return this.$store.getters['RiverEvents/releasesFromTodayThenPast']
     },
+
     editBetaBoxKey () {
       return `editBetaBox${this.reachId}`
     },
     reachId () {
       return this.$route.params.id
     },
-    reachGage() {
+    reachGage () {
 
       if (this.river && this.river.readingsummary && this.gages) {
-        return this.gages.find(g => g.gauge.id.toString() === this.river.readingsummary.gauge_id.toString());
+        return this.gages.find(g => g.gauge.id.toString() === this.river.readingsummary.gauge_id.toString())
       }
       return this.gages[0]
     }
   },
   methods: {
-    getReleaseFieldLabel(releaseDate) {
-      if(!releaseDate) return ''
+    /**
+     *
+     * @param releaseDate - string e.g. "2007-01-01T11:33:00+0"
+     * @return "Latest" or "Next"
+     */
+    getReleaseFieldLabel (releaseDate) {
+      if (!releaseDate) return ''
 
       const today = new Date()
       const eventDate = new Date(releaseDate)
```

Comment:
is this going to break (currently broken) on Safari? we've been getting a bunch of tricky Safari issues related to Date objects, I know we're not using `Date.parse` here but I'm still not certain

Reply:
to your point, I think it's safer to use a string compare, but safari seemed to be chugging along ok on the input.